client/state/login/magic-login/actions.js

```diff
@@ -92,6 +88,28 @@ export const fetchMagicLoginRequestEmail = ( email, redirectTo ) => dispatch =>
 		} );
 };
 
+class MagicLoginError extends Error {
```

Comment:
Is this class needed at all? The only field we ever use is `status`, and that's present on the original fetch `Response`, too.

It would be a good idea to use the same `HTTPError` or `LoginError` as for the regular login. The errors aren't that different.

Reply:
Right, that's a good point. It's more complex, but it serves us well here too.